pkgs/development/python-modules/shapely/default.nix

```diff
@@ -1,27 +1,28 @@
 { lib
 , stdenv
 , buildPythonPackage
-, pythonOlder
 , fetchPypi
+, pytestCheckHook
+, pythonOlder
+
 , cython
 , geos
+, numpy
 , oldest-supported-numpy
 , setuptools
 , wheel
-, numpy
-, pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "shapely";
-  version = "2.0.2";
+  version = "2.0.3";
   format = "pyproject";
```

Comment:
```suggestion
  pyproject = true;
```

Reply:
Thank you, good point, applied in 391634c3434fd98f708b78a9339efc6149366e86 .